src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/OgcXmlUtil.java

```diff
@@ -0,0 +1,130 @@
+package de.terrestris.shogun2.util.interceptor;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.Charset;
+
+import javax.servlet.ServletInputStream;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.springframework.util.StreamUtils;
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+/**
+ *
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class OgcXmlUtil {
+
+	/**
+	 * The Logger.
+	 */
+	private static final Logger LOG = Logger.getLogger(OgcXmlUtil.class);
+
+	/**
+	 *
+	 * @param request
+	 * @return
+	 */
+	public static String getRequestBody(MutableHttpServletRequest request){
```

Comment:
I think it would allow for easier reuse if this accepted `HttpServletRequest`, not our very specific `MutableHttpServletRequest`,. Do you see any problems with that change?